tests/components/homekit_controller/test_sensor.py

```diff
@@ -76,3 +87,18 @@ def create_light_level_sensor_service():
     helper.characteristics[LIGHT_LEVEL].value = 20
     state = await helper.poll_and_get_state()
     assert state.state == "20"
+
+
+async def test_carbon_dioxide_level_sensor_read_state(hass, utcnow):
+    """Test reading the state of a HomeKit carbon dioxide sensor accessory."""
+    sensor = create_carbon_dioxide_level_sensor_service()
+    helper = await setup_test_component(
+        hass, [sensor], suffix="carbon_dioxide_level")
```

Comment:
Black wants you to do this:

```
    helper = await setup_test_component(hass, [sensor], suffix="carbon_dioxide_level")
```

(I.e. all on one line)

Reply:
oh okay.. yesterday it failed due to the line being more than 79 characters :)